crates/uv/src/main.rs

```diff
@@ -1012,10 +1016,17 @@ async fn run() -> Result<ExitStatus> {
                 Vec::new(),
                 args.no_index,
             );
+            let cwd = env::current_dir()?;
```

Comment:
We should probably avoid getting the cwd if the prompt is not set to `.`

Reply:
I am totally newbie to Rust so I don't know how to handle this lifetime issue.

```rust
            let prompt = if args.prompt == "." {
                match env::current_dir() {
                    Ok(cwd) => match cwd.file_name() {
                        Some(b) => b.to_str(),
                        None => None,
                    },
                    Err(_) => None,
                }
            } else {
                Some(args.prompt.as_str())
            };
```

```
error[E0597]: `cwd` does not live long enough
    --> crates/uv/src/main.rs:1021:38
     |
1019 |             let prompt = if args.prompt == "." {
     |                 ------ borrow later stored here
1020 |                 match env::current_dir() {
1021 |                     Ok(cwd) => match cwd.file_name() {
     |                        ---           ^^^ borrowed value does not live long enough
     |                        |
     |                        binding `cwd` declared here
...
1024 |                     },
     |                     - `cwd` dropped here while still borrowed
```